src/SupabaseClient.ts

```diff
@@ -100,7 +100,7 @@ export default class SupabaseClient<
       this.headers,
       settings.global.fetch
     )
-    this.fetch = fetchWithAuth(supabaseKey, this._getAccessToken.bind(this), settings.global.fetch)
```

Comment:
Removing `settings.global.fetch` denies developers the ability use a custom fetch, passed into the Supabase client's options.

Reply:
Reverted! Thank you.